app/src/main/java/com/nutomic/syncthingandroid/views/FoldersAdapter.java

```diff
@@ -40,7 +40,7 @@ public class FoldersAdapter extends ArrayAdapter<Folder> {
 
     private static final String TAG = "FoldersAdapter";
 
-    private final HashMap<String, FolderStatus> mLocalFolderStatuses = new HashMap<>();
+    public final HashMap<String, FolderStatus> mLocalFolderStatuses = new HashMap<>();
```

Comment:
The `m` prefix is used for private members. I think the javay thing to do is to keep it private and add a getter. Maybe a useful interface would be `getLocalFolderStatus(String folderId)` as the use in this PR only cares about a single folder.

Reply:
implementing that public interface...